### PR TITLE
Add braces

### PR DESCRIPTION
**Pull request by NextTurn**
Contributes to #1884 

Enforces [SA1520](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1520.md).

### CODE REVIEW

File: snippets/csharp/VS_Snippets_CFX/c_customauthpol/cs/c_customauthpol.cs [hunk 1]
```diff
@@ -67,10 +67,12 @@ public bool Evaluate(EvaluationContext evaluationContext, ref object state)
 		state = customstate;
 	  }
 	  else
-		customstate = (CustomAuthState)state;
+            {
+                customstate = (CustomAuthState)state;
+            }
```
Comment (Youssef1313) [inline snippets/csharp/VS_Snippets_CFX/c_customauthpol/cs/c_customauthpol.cs]: Indentation needs to be fixed.

File: snippets/csharp/VS_Snippets_CFX/c_customauthpol/cs/c_customauthpol.cs [hunk 2]
```diff
@@ -67,10 +67,12 @@ public bool Evaluate(EvaluationContext evaluationContext, ref object state)
...
 
-	  // If claims have not been added yet...
-	  if (!customstate.ClaimsAdded)
+            // If claims have not been added yet...
+            if (!customstate.ClaimsAdded)
 	  {
```
Comment (Youssef1313) [inline snippets/csharp/VS_Snippets_CFX/c_customauthpol/cs/c_customauthpol.cs]: Indentation is bad here too.

File: snippets/csharp/VS_Snippets_CFX/c_customwsdlextensions/cs/datacontractdocumentation.cs [hunk 3]
```diff
@@ -38,8 +38,10 @@ object[] dcDocAttrs
           + "</summary>";
       }
       else
-        return null;
-    }
+            {
+                return null;
+            }
```
Comment (Youssef1313) [inline snippets/csharp/VS_Snippets_CFX/c_customwsdlextensions/cs/datacontractdocumentation.cs]: Indentation is bad here too.

File: snippets/csharp/VS_Snippets_CFX/c_customauthpol/cs/c_customauthpol.cs (hunk 1 above)
Comment (NextTurn) [reply snippets/csharp/VS_Snippets_CFX/c_customauthpol/cs/c_customauthpol.cs]: To keep this PR simple, I'm not making extra changes.

File: snippets/csharp/VS_Snippets_CFX/c_customwsdlextensions/cs/datacontractdocumentation.cs [hunk 4]
```diff
@@ -53,8 +55,10 @@ object[] dcDocAttrs
...
```
Comment (Youssef1313) [inline snippets/csharp/VS_Snippets_CFX/c_customwsdlextensions/cs/datacontractdocumentation.cs]: and here

File: snippets/csharp/VS_Snippets_CFX/dcacontainernovalidation/cs/client.cs [hunk 5]
```diff
@@ -42,9 +42,11 @@ public static void Main()
...
             }
             else
-              Console.WriteLine("Any extra data: {0}.", replyPerson.additionalData.ToString());
+                {
+                    Console.WriteLine("Any extra data: {0}.", replyPerson.additionalData.ToString());
+                }
 
-            if (replyPerson.Blob != null)
+                if (replyPerson.Blob != null)
               Console.WriteLine("And the added object is: " + replyPerson.Blob.ToString());
```
Comment (Youssef1313) [inline snippets/csharp/VS_Snippets_CFX/dcacontainernovalidation/cs/client.cs]: and here.

File: snippets/csharp/VS_Snippets_CLR/CryptoWalkThru/cs/Form1.cs [hunk 6]
```diff
@@ -78,8 +78,10 @@ private void buttonCreateAsmKeys_Click(object sender, System.EventArgs e)
 private void buttonEncryptFile_Click(object sender, System.EventArgs e)
 {
     if (rsa == null)
-        MessageBox.Show("Key not set.");
-    else
+            {
+                MessageBox.Show("Key not set.");
+            }
+            else
     {
```
Comment (Youssef1313) [inline snippets/csharp/VS_Snippets_CLR/CryptoWalkThru/cs/Form1.cs]: and here. It seems that there will be more occurrences of the indentation issue. I'm not going to point more of them.

**Comment by NextTurn** [comment]
To keep this PR simple, I'm not making extra changes.

File: snippets/csharp/VS_Snippets_CLR/IndentedTextWriterExample/CS/form1.cs [hunk 7]
```diff
@@ -55,10 +55,12 @@ private void WriteLevel(IndentedTextWriter indentWriter, int level, int totalLev
                 indentWriter.Indent--;
             }
             else
+            {
                 //<Snippet6>
                 // Outputs a string using the WriteLineNoTabs method.
                 indentWriter.WriteLineNoTabs("This is a test phrase written with the IndentTextWriter.WriteLineNoTabs method.");
-                //</Snippet6>
+            }
+            //</Snippet6>
```
Comment (Youssef1313) [inline snippets/csharp/VS_Snippets_CLR/IndentedTextWriterExample/CS/form1.cs]: The snippet closing tag should be moved **inside** the else clause.

**Comment by BillWagner** [comment]
Thanks for doing this work @NextTurn 

As @Youssef1313 points out, many of these samples do have inconsistent indentation. I agree that these should be fixed, and should be handled in a separate PR.

**Pull request merged**